src/webdriver_auto_update/webdriver_auto_update.py

```diff
@@ -74,8 +75,12 @@ def obtain_os(self):
         Returns:
             str: The operating system identifier.
         """
-        if sys.platform.startswith('win32') or sys.platform.startswith('cygwin'):
-            return 'win32'
+        bit_architecture = platform.architecture()[0]
+        bit_number = re.findall(r"\d+", bit_architecture)
+        os_bit = bit_number[0] if bit_number else "Unknown"
```

Comment:
Currently the os_bit variable is used in line 83 to obtain the chromedriver platform for windows.
`return 'win' + os_bit`

I would suggest to change line 80 to the following so that at least the operating system can support running the 32-bit option:
`os_bit = bit_number[0] if bit_number else "32"`.